moonbeam-types-bundle/index.ts

```diff
@@ -451,6 +451,118 @@ export const moonbeamDefinitions = {
         },
       },
     },
+    {
+      minmax: [38, undefined],
+      types: {
+        AccountId: "EthereumAccountId",
+        AccountInfo: "AccountInfoWithTripleRefCount",
+        Address: "AccountId",
+        AuthorId: "AccountId",
```

Comment:
this needs to be RelayChainAccountId

Reply:
I copied the types from the previous version and added `ProxyType`. There should be no other changes in the bundle right ?

Reply:
yes indeed but I know this change is necessary for the author mappping stuff so might as well put it here to make sure we don't get unrelated errors